core/scheduler.py

```diff
@@ -3,24 +3,24 @@
 import json
 from collections import deque
 
+import ray
 import schedule
 
 from core.utils.dmljob import DMLJob
-from core.runner import DMLRunner
+from core.runner_actor import DMLRunnerActor
 
+
+ray.init()
 logging.basicConfig(level=logging.DEBUG,
                     format='[Scheduler] %(asctime)s %(levelname)s %(message)s')
 
 class DMLScheduler(object):
```

Comment:
Write docstrings for each method. Some methods have comments right before their definition. These comments should be moved to the docstrings or inside the method definition.

Reply:
Done.